contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.3.0--4.4.0.sql

```diff
@@ -11,6 +11,224 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
  * final behaviour.
  */
 
+do
+LANGUAGE plpgsql
+$$
+DECLARE securityadmin TEXT;
+BEGIN
+    IF EXISTS (
+               SELECT FROM pg_catalog.pg_roles
+               WHERE  rolname = 'securityadmin') 
+        THEN
+            RAISE EXCEPTION 'Role "securityadmin" already exists.';
```

Comment:
This exception will fail Latest-latest upgrade.

Reply:
fixed roles are only targeted for PG 16, so should not be problem right ?

Reply:
yes